sim-administration/postgres/README.md

```diff
@@ -0,0 +1,20 @@
+# SIM Manager DB schema
```

Comment:
Please take a look at  the Codacy suggestions for the markdown file.  They are minor, but will probably make the markdown a little stricter.